emf/src/main/java/io/lionweb/lioncore/java/emf/EMFMetamodelImporter.java

```diff
@@ -146,17 +146,45 @@ private void processStructuralFeatures(
     for (EStructuralFeature eFeature : eClass.getEStructuralFeatures()) {
       if (eFeature.eClass().getName().equals(EcorePackage.Literals.EATTRIBUTE.getName())) {
         EAttribute eAttribute = (EAttribute) eFeature;
-        Property property = new Property(eFeature.getName(), classifier);
-        property.setID(ePackage.getName() + "-" + classifier.getName() + "-" + eFeature.getName());
-        property.setKey(ePackage.getName() + "-" + classifier.getName() + "-" + eFeature.getName());
-        classifier.addFeature(property);
-        property.setOptional(!eAttribute.isRequired());
-        DataType<DataType> propertyType =
-            dataTypeMapping.convertEClassifierToDataType(eFeature.getEType());
-        Objects.requireNonNull(propertyType, "Cannot convert type " + eFeature.getEType());
-        property.setType(propertyType);
-        if (eAttribute.isMany()) {
-          throw new IllegalArgumentException("EAttributes with upper bound > 1 are not supported");
+        // EAttributes with upper bound > 1 are not supported in LionWeb
```

Comment:
I think this comment doesn't need to be there: you're saying the same thing on line 161 already, and you first handle the upper bound <= 1 case.